### DISCUSSION TITLE
Log user ID in Moderation -> Logs

### DISCUSSION THREAD

**Discussion by D3XX3R** [Chatterino/chatterino2#3194] [Ideas] [answered]
I'm using the logging function of Chatterino all the time and the only feature missing for it is logging user IDs.
If a user changes it's name it's hard to retrace it's old names and messages attached to it. But if the logs would contain the user ID it would be easier to search for old messages written by said user.

**Comment by Felanbird** [comment, marked as answer]
#1799
